pkg/keystone/jobs.go

```diff
@@ -0,0 +1,185 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package keystone
+
+import (
+	keystonev1 "github.com/openstack-k8s-operators/keystone-operator/api/v1beta1"
+	keystonev1beta1 "github.com/openstack-k8s-operators/keystone-operator/api/v1beta1"
+
+	common "github.com/openstack-k8s-operators/lib-common/pkg/common"
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// BootstrapJob func
```

Comment:
Moving these functions into jobs.go in a follow up would make the changes here a lot easier to review. Essentially keep BootstrapJob in bootstrap.go and DbSyncJob in dbsync.go. I'd be fine to eventually move them but with patches like this minimizing structural moves like this is beneficial I think.

Reply:
done